### ISSUE TITLE
Plugins on Raspberry Pi

### ISSUE THREAD

**Issue by FidesTorres** [ajenti/ajenti#1408]
The Plugins don't work on Raspberry Pi. Error: The URL http://<ip>:8000/view/plugins/ was not found on this server.

**Comment by kiarn** [comment]
Yes, testing on RPi is still on my todo list, raspian is actually not supported, but I want to solve this.

Arnaud
